src/docs/deployment/ios.md

```diff
@@ -89,10 +89,10 @@ In the **Identity** section:
 : Runner
 
 {{site.note.alert}}
```

Comment:
The macro is:

```
{{site.alert.note}}
   xxxx
{{site.alert.end}}
```

Reply:
Well that would do it.
https://github.com/flutter/website/pull/3838/files#diff-e8a9c09b65f16491acbbde77bcbf4883R94